DashWallet/Sources/UI/Coinbase/Base/UIViewController+Coinbase.swift

```diff
@@ -19,10 +19,15 @@ import UIKit
 
 extension BaseViewController {
     public func showSuccessTransactionStatus(text: String) {
-        let vc = SuccessfulOperationStatusViewController.initiate(from: sb("Coinbase"))
+        let vc = SuccessfulOperationStatusViewController.initiate(from: sb("OperationStatus"))
         vc.closeHandler = { [weak self] in
             guard let wSelf = self else { return }
-            wSelf.navigationController?.popToViewController(wSelf.previousControllerOnNavigationStack!, animated: true)
```

Comment:
It doesn't look like `closeHandler` was actually invoked in the `SuccessfulOperationStatusViewController` before. I moved the code from action handler in here.